src/components/sdk/MeetingControls/AudioInputVFControl.tsx

```diff
@@ -99,6 +102,18 @@ const AudioInputVFControl: React.FC<Props> = ({
     }
   }, [device]);
 
+  useEffect(() => {
+    if (!audioVideo) {
+      return;
+    }
+    if (device instanceof VoiceFocusTransformDevice) {
+      if (isVoiceFocusEnabled) {
+        console.info('VoiceFocusTransformDevice observeMeetingAudio.');
```

Comment:
You probably want to add which device you are adding the meeting audio observer to help while debugging.

```
console.info(`Add observeMeetingAudio on ${JSON.stringify(device)} VoiceFocusTransformDevice.`);
```

Reply:
doesnt the meetingManger have a logger reference instead of using console.info?